### DISCUSSION TITLE
Error when transferring Akropolis ERC-20 to SPL through Wormhole (unknown token)

### DISCUSSION THREAD

**Discussion by gowpow** [wormhole-foundation/wormhole#302] [Problems / bugs] [answered]
I attempted to transfer my ERC-20 Akropolis over to Solana using Wormhole on June 20, 2021; however, the transaction encountered an error. My Akropolis ERC-20 was removed from my Ether wallet and I received an unknown token in my Solana wallet.

Can someplace help me figure out what went wrong? IS there a way to resolve the error?

Below are the details:

Transaction: 5XJBaYLW126GWKcJrqN8NX6XqzspBnSAjjocrcqtYJuoD6qQho1rKiesgn7jJnsi8UN9i6FD1UgJTdARcyZGjsuy

Program WormT3McKhFJ2RkiGpdw9GKvNCrB2aB54gb2uV9MfQC invoke [1]
Instruction: PostVAA
Error: InvalidArgument
Program WormT3McKhFJ2RkiGpdw9GKvNCrB2aB54gb2uV9MfQC consumed 75427 of 200000 compute units
Program WormT3McKhFJ2RkiGpdw9GKvNCrB2aB54gb2uV9MfQC failed: invalid program argument
Transaction failed: Error processing Instruction 0: invalid program argument

**Comment by leoluk** [comment, marked as answer]
The "unknown token" you received is the wrapped token. Unfortunately, SPL tokens don't have on-chain names on Solana so it shows as "unknown" in your wallet unless it's part of a hard-coded list of token names in the wallet. I would assume that "wrapped Akropolis" is not part of this list of well-known tokens.

**Reply by classicsinatra** [reply]
And for the other way around? I have a registered token on Solana, need to get this contract registered so it is not an unknown wormhole token on ETH: https://etherscan.io/token/0x3EF389f264e07fFF3106A3926F2a166d1393086F?a=0x5c95d9bff0f672562df784a5787eea621a01dbdb

**Reply by leoluk** [reply]
It's the same way the other way. If there's no name on Solana there won't be a name on ETH, either.
